app/controllers/two_factor_authentication/totp_verification_controller.rb

```diff
@@ -4,6 +4,8 @@ class TotpVerificationController < ApplicationController
 
     skip_before_action :handle_two_factor_authentication
 
+    helper_method :confirmation_for_phone_change?
```

Comment:
Why does the TOTP verification controller need to know about phone change confirmation? That only applies to OTP verification. You can only confirm a phone by receiving an SMS.